yihan correct text on info modal on the Projects page

AriaYu927 opened the pull request:
# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/94303659/251204a8-16a9-49bf-a737-5d1f83d07f6f)

## Main changes explained:
- Update file projects.css for adding padding-left property on `ol` element.

## How to test:
1. check into current branch
2. do `npm run start:local` to run this PR locally
3. log as admin user
4. go to Other Links > Projects > Projects header (i) button
5. verify the text is properly aligned and hangs off left edge of modal.

## Screenshots or videos of changes:
![Visibility_issue](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/94303659/b795dc75-ff9d-4602-8654-f396f17da9fa)

OAkinbode reviewed (dismissed):
Hello Yihan, 

The text presentation 
<img width="1440" alt="PR 898 working" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104801652/c44b2cc3-8519-4e0d-91e5-8b5b88946a26">
looks good to me.

AuroraLovesWhales reviewed (approved):
Hi @AriaYu927. I have reviewed your PR and the change works as intended. Good job!
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/44281916/db0f5bf4-f96f-49b2-99d2-582b7585a93c)

KurtisIvey reviewed (approved):
Looks way better now. Great job 

![pr898Fixed](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/19482363/6bd018d9-591d-4ad8-bf0a-e8b6184cd635)

stevenpaalz reviewed (approved):
Looks much better now. Thanks!
![Screenshot 2023-06-08 at 2 19 01 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/121642083/37d7730b-f6fc-4158-9390-4b2cb671dd84)

leonzh2k reviewed (changes requested):
Looks like the intended functionality has been achieved:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49804462/71ce6fc1-1847-4e69-9853-b4eefacf4e6d)

But there are unintended side effects. Notice that you are using a globally scoped selector to select all `<ol>` elements. This means that this styling will affect all `<ol>` elements in every component, which we don't want. To confirm that this would happen, I added an `<ol>` to a different component (`src/components/LeaderBoard/Leaderboard.jsx`) and compared the styling before and after your changes: 

Before:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49804462/a20fe9d7-261f-4174-a153-8dc84c911638)

After:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49804462/f244e1fd-f70d-4582-8dcd-b5174216f2cd)

You could see how this global styling could cause confusion in the future; a developer would be wondering why the styling for a new `<ol>` they added isn't working, when the cause is in this file. I'd say to never use global selectors unless you really want the styles to apply globally (in that case, those styles should go into some sort of `globals.css` file). I strongly recommend making changes so that the styling only applies to this project info modal.

20chen-7 reviewed (dismissed):
Looks as intended
![Screenshot 2023-06-08 at 2 30 49 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/66c0b5f2-234c-4197-866f-673cc1b15e4f)

bienzguoa reviewed (approved):
Looks good! It works as intended.
<img width="1121" alt="Screenshot 2023-06-08 at 9 42 21 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92126559/c22fccac-e437-4ed9-a9bf-227aeab9014d">

eduhorta reviewed (approved):
Looking good! Nice job! 👍

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/c3273274-286e-4241-b121-0013869c7531)

cvtqx reviewed (approved):
The text is aligned well.
<img width="969" alt="Screen Shot 2023-06-09 at 2 10 58 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/107726913/8b57ceee-6504-4d58-96fc-6f2eaceae2b7">

AriaYu927 commented:
> Looks like the intended functionality has been achieved: ![image](https://user-images.githubusercontent.com/49804462/244477678-71ce6fc1-1847-4e69-9853-b4eefacf4e6d.png)
> 
> But there are unintended side effects. Notice that you are using a globally scoped selector to select all `<ol>` elements. This means that this styling will affect all `<ol>` elements in every component, which we don't want. To confirm that this would happen, I added an `<ol>` to a different component (`src/components/LeaderBoard/Leaderboard.jsx`) and compared the styling before and after your changes:
> 
> Before: ![image](https://user-images.githubusercontent.com/49804462/244482824-a20fe9d7-261f-4174-a153-8dc84c911638.png)
> 
> After: ![image](https://user-images.githubusercontent.com/49804462/244483154-f244e1fd-f70d-4582-8dcd-b5174216f2cd.png)
> 
> You could see how this global styling could cause confusion in the future; a developer would be wondering why the styling for a new `<ol>` they added isn't working, when the cause is in this file. I'd say to never use global selectors unless you really want the styles to apply globally (in that case, those styles should go into some sort of `globals.css` file). I strongly recommend making changes so that the styling only applies to this project info modal.

Hi @leonzh2k , thanks for your reviewing. I've updated the code based on your suggestion, please check it again.

AnishPandita commented:
Hi @AriaYu927 I tested the PR and it seems to work well. The text alignment looks much better now. Great Work.
<img width="1440" alt="Screenshot 2023-06-09 at 4 43 38 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/111321945/f6da29e7-cd19-42c3-9475-c60a2e4140fc">

nathanah reviewed (approved):
Looks good to me.
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/786e1f25-50b6-470e-a6d0-df309636b242)

mahamakhan reviewed (approved):
Looks perfect to me.
<img width="257" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/106139234/44b3ecb3-102c-467d-82c1-74365e14c06d">

yufeiStella reviewed (approved):
It works well for me.
<img width="670" alt="Screen Shot 2023-06-09 at 23 03 23" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/111897624/e20a45a7-3e54-45e2-bcd9-12a7ac6fd40b">

leonzh2k reviewed (approved):
@AriaYu927 Looks like the scoping issue has been fixed. Only thing I'm not sure about is if the naming of this selector conforms to the rest of the codebase. Looks like BEM naming conventions have been used historically, but it's applied pretty inconsistently. I approve, but it's up to you if you want to change the naming.

rachnapandya commented:
Looks good to me after the changes.
![review1](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/53634437/72f3f3ec-136a-4c61-a5d0-00322ec834a3)

EdwinLauM reviewed (approved):
Changes look good and works as intended! Good job! 

<img width="1374" alt="Screen Shot 2023-06-10 at 7 21 27 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/42288987/6cb1ba72-3e0d-41f7-8236-8943ad060f47">

one-community commented:
Thank you all. Merging!
